testProjectAndroid/build.gradle

```diff
@@ -105,70 +123,70 @@ afterEvaluate {
   }
 
   test.doLast {
-    org.junit.Assert.assertEquals(
-        ['generateArmFreeappDebugAndroidTestProto', 'generateArmFreeappDebugProto',
+    assert ['generateArmFreeappDebugAndroidTestProto', 'generateArmFreeappDebugProto',
          'generateArmFreeappReleaseProto', 'generateArmRetailappDebugAndroidTestProto',
          'generateArmRetailappDebugProto', 'generateArmRetailappReleaseProto',
          'generateX86FreeappDebugAndroidTestProto', 'generateX86FreeappDebugProto',
          'generateX86FreeappReleaseProto', 'generateX86RetailappDebugAndroidTestProto',
-         'generateX86RetailappDebugProto', 'generateX86RetailappReleaseProto'] as Set,
-         protobuf.generateProtoTasks.all().collect({ it.name }) as Set)
-    org.junit.Assert.assertEquals(
-        ['generateArmFreeappDebugAndroidTestProto',
+         'generateX86RetailappDebugProto', 'generateX86RetailappReleaseProto'] as Set ==
+         protobuf.generateProtoTasks.all().collect({ it.name }) as Set
+
+    assert ['generateArmFreeappDebugAndroidTestProto',
```

Comment:
Hi, I am not sure what do you mean by `whether it's enable or not` but groovy assert is always executed, and provide much better comparison report
see http://groovy-lang.org/semantics.html#_power_assertion